src/EditorFeatures/Core/xlf/EditorFeaturesResources.cs.xlf

```diff
@@ -2,6 +2,16 @@
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
   <file datatype="xml" source-language="en" target-language="cs" original="../EditorFeaturesResources.resx">
     <body>
+      <trans-unit id="Add_Missing_Imports_On_Paste">
+        <source>Add Missing Imports On Paste</source>
+        <target state="new">Add Missing Imports On Paste</target>
```

Comment:
this should likely be a different string per language.  For # it should be `Add missing 'usings' on paste`.  For VB it should be `Add missing 'Imports' on paste`

Reply:
This is for the paste handler, which afaict can not have a variable name based on the context of the command.

Reply:
> This is for the paste handler, which afaict can not have a variable name based on the context of the command.

it def can.  it just needs to eitehr be abstract, with C#/Vb subclasses.  or it should get the name from the interface it calls into.

Reply:
That would mean we're exporting two command handlers right? How does that work in chained ordering?

Reply:
With different content type this makes sense to me. Changed.